sdk/messaging/azservicebus/go.mod

```diff
@@ -3,21 +3,20 @@ module github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus
 go 1.16
 
 require (
-	github.com/Azure/azure-amqp-common-go/v3 v3.1.0
+	github.com/Azure/azure-amqp-common-go/v3 v3.2.0
 	github.com/Azure/azure-sdk-for-go v51.1.0+incompatible
 	github.com/Azure/azure-sdk-for-go/sdk/internal v0.7.0
 	github.com/Azure/go-amqp v0.15.0
 	github.com/Azure/go-autorest/autorest v0.11.18
```

Comment:
These `go-autorest` dependencies will be gone at some point I assume.

Reply:
We do have an HTTP client in here (it's for the ATOM endpoint that lets you manage Service Bus resources). 

Would we still remove it then? I'm not familiar with the package deps for HTTP stacks in Go.

Reply:
Yep, this is all track 1 artifacts that need to go.

Reply:
Okay, cool. Yeah, I'll see about removing this in a future PR. I might have to push till after the first beta.

Reply:
If you just need an HTTP client you can use `net/http.DefaultClient` from the standard library.